pages/bitmark-references/node-setup/bitmark-node-setup.md

```diff
@@ -9,154 +9,116 @@ folder: bitmark-references/node-setup
 
 This tutorial explains how to set up the `bitmarkd` and/or the `recorderd` services by hand.
 
 ## Prerequisites
 
-* You must install the go language package for your system.
-* You must configure environmental variables for the go system.
-* You must install the ZMQ4 and Argon2 libraries.
-
-## Installing the Prerequisite Packages
++ You must have and know your **public IP** address.
++ You must have **a Bitcoin and a Litecoin address** for receiving mining rewards.
++ Your environment must be open for incoming connections through port 2136.
++ Hardware minimum requirements:
+    - memory >= 2 GB
+    - free disk space >= 20 GB
+    - broadband Internet connection with upload speeds of at least 400 kilobits (50 kilobytes) per second
 
-Installing the required packages requires specific commabds on Debian, FreeBSD, macOS, and Ubuntu.
+## Installing a Bitmark Node and Generating Required Keys and Certificates
 
-### Installing Packages on Debian
+A Bitmark node can be installed differently, depending on the operating system. Installations on FreeBSD, Ubuntu, and macOS can be done through pre-built packages.
 
-_Tested on Stretch (Debian 9)._
+If some special functionality is required, or there are no available packages yet for your OS, a Bitmark node can also be compiled from source files. For more information, see [Building From Source](###-Building-From-Source).
 
-First, ensure that the system has access to both the current version of Debian (`stable`) and `testing`.
-~~~
-root@debian-bitmarkd:/# cat /etc/apt/sources.list.d/stable.list
-deb     http://ftp.de.debian.org/debian/    stable main contrib non-free
-deb-src http://ftp.de.debian.org/debian/    stable main contrib non-free
-deb     http://security.debian.org/         stable/updates  main contrib non-free
+### FreeBSD
 
-root@debian-bitmarkd:/# cat /etc/apt/sources.list.d/testing.list
-deb     http://ftp.de.debian.org/debian/    testing main contrib non-free
-deb-src http://ftp.de.debian.org/debian/    testing main contrib non-free
-deb     http://security.debian.org/         testing/updates  main contrib non-free
-~~~
-
-Now. install `libargon2` from `testing`:
 ```shell
-    $ apt-get -t testing install libargon2-dev libargon2-1
+$ pkg install bitmark
+$ service bitmarkd keygen
+$ service recorderd keygen
 ```
 
-For the other packages, install from `stable` or `testing` as you prefer; both versions work:
-```shell
-    $ apt install uuid-dev libzmq3-dev
-```
-### Installing Packages on FreeBSD
+### Ubuntu
 
 ```shell
-    $ pkg install libzmq4 libargon2
+$ add-apt-repository -y ppa:bitmark/bitmarkd
+$ apt update
+$ apt install bitmarkd
 ```
 
-### Installing Packages on MacOSX
+### macOS
 
-_Be sure that homebrew is installed correctly before beginning._
+You need [Brew](https://brew.sh/) installed and configured.
 
```

Comment:
If we're going for simplicity and accessibility, it might be good to include the precise Brew commands here:

You need to install Brew:
```
/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```
Visit the [Brew](https://brew.sh/) site for more information on the installation and for configuration options.